content/en/tracing/llm_observability/sdk.md

```diff
@@ -265,19 +328,23 @@ The `LLMObs.annotate()` method accepts the following arguments:
 
 `input_data` 
 : optional - _JSON serializable type or list of dictionaries_ 
-<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans).
+<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans).  **Note**: Embedding spans are a special case and require a string, dictionary, or a list of dictionaries with this format: `{"text": "...", "name": "...", "score": float, "id": "..."}`.
 
 `output_data` 
 : optional - _JSON serializable type or list of dictionaries_ 
-<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans). **Note**: Retrieval spans are a special case and require a list of dictionaries with this format: `{"text": "...", "name": "...", "score": float, "id": "..."}`.
+<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans). **Note**: Retrieval spans are a special case and require a string, dictionary, or a list of dictionaries with this format: `{"text": "...", "name": "...", "score": float, "id": "..."}`.
 
 `metadata` 
 : optional - _dictionary_
 <br />A dictionary of JSON serializable key-value pairs that users can add as metadata information relevant to the input or output operation described by the span (`model_temperature`, `max_tokens`, `top_k`, and so on).
 
-`tags` 
+`metrics`
 : optional - _dictionary_
-<br />A dictionary of JSON serializable key-value pairs that users can add as tags regarding the span's context (`session`, `environment`, `system`, `versioning`, and so on).
+<br />A dictionary of JSON serializable keys and numeric values that users can add as metrics relevant to the operation described by the span (`prompt_tokens`, `completion_tokens`, `total_cost`, and so on).
```

Comment:
Should be `input_tokens`, `output_tokens`, `total_tokens`

to standardize across different span kinds and model providers